test/expect-tests/dune_action/dune

```diff
@@ -0,0 +1,16 @@
+(data_only_dirs foo_dir)
+(copy_files foo_dir/foo)
```

Comment:
I don't think you need `copy_files` if you're accessing `foo_dir/foo` directly. I think `(copy_files glob)` is conceptually the same as `cp $glob ./`, but lazy (the files will be only copied if they're needed).

Reply:
True, it was not needed.